UI/importers/sl.cpp

```diff
@@ -503,11 +503,17 @@ bool SLImporter::Check(const string &path)
 OBSImporterFiles SLImporter::FindFiles()
 {
 	OBSImporterFiles res;
-#ifdef _WIN32
+#if defined(_WIN32) || defined(__APPLE__)
 	char dst[512];
 
 	int found = os_get_config_path(dst, 512,
-				       "slobs-client\\SceneCollections\\");
+#if defined(_WIN32)
+				       "slobs-client\\SceneCollections\\"
+#elif defined(__APPLE__)
+				       "slobs-client/SceneCollections/"
+#endif
```

Comment:
Hmm, do we not normalise paths at any point, removing the need for redefining the same string with different slashes?

Reply:
Yes, and it appears the original implementation didn't use it.
I fixed that (and confirmed Windows still works).